Mathlib/Analysis/Complex/IsIntegral.lean

```diff
@@ -0,0 +1,27 @@
+/-
+Copyright (c) 2022 Yuyang Zhao. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yuyang Zhao
```

Comment:
Is this a correct header? If yes, then please add a `Co-authored-by` to the top comment (a.k.a. commit message). Otherwise LGTM.
bors d+

P.S.: Idea for another PR: prove that all Gaussian integers are integral (in `Complex` or in `GaussianInt`, or both).